middle_end/flambda2/simplify/simplify_let_cont_expr.ml

```diff
@@ -833,9 +981,11 @@ let get_uses (data : after_downwards_traversal_of_body_and_handlers_data) cont =
       Continuation.print cont
   | Some cont -> cont
 
-let create_handler_to_rebuild
-    (data : after_downwards_traversal_of_body_and_handlers_data) cont
+let create_handler_to_rebuild cont_lifting_params
+    (data : after_downwards_traversal_of_body_and_handlers_rebuild) cont
     (handler : handler_after_downwards_traversal) =
+  (* See comment at the top of
+     [after_downwards_travsersal_of_body_and_handlers]. *)
```

Comment:
The comment is not at the top of `after_downwards_travsersal_of_body_and_handlers` now